frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -217,15 +220,15 @@ const ReviewPage = () => {
             <Tooltip
               hasArrow
               label={REVIEW_PAGE_TOOL_TIP_COPY}
-              isDisabled={stage === "REVIEW"}
+              isDisabled={isDisabled}
```

Comment:
I think this should be `!isDisabled`

Reply:
<img width="1816" alt="Screen Shot 2022-01-02 at 10 40 25 PM" src="https://user-images.githubusercontent.com/37675216/147898233-247a74f2-47bb-4c37-8c8e-4346cd94c20a.png">